encode_test.go

```diff
@@ -304,24 +304,29 @@ func TestInfinityTimestamp(t *testing.T) {
 	var err error
 	var resultT time.Time
 
-	expectedErrorStrPrefix := `sql: Scan error on column index 0: unsupported`
+	expectedErrorStrRegexp, err := regexp.Compile(
```

Comment:
regexp.MustCompile is fine here since it should always succeed

Reply:
Sure, I'll update the PR shortly